src/app/pages/PerpetualPage/hooks/usePerpetual_OrderHistory.ts

```diff
@@ -134,7 +129,7 @@ export const usePerpetual_OrderHistory = (
             );
             return {
               id: item.id,
-              pairType: pairType,
+              pair: getPairByPerpId(item?.perpetual?.id),
```

Comment:
```suggestion
              pair: PerpetualPairDictionary.getById(item?.perpetual?.id),
```

Reply:
Actually, you should declare it at the top, use `useMemo` and use the value here and on line `104` as well.